moddio/taro2
# fix: update scale ratio for box2dwasm

## Summary
- box2dwasm have velocity limit (2 unit per minutes), so I had to scale down the physics bodies' size 
- it still have some minor differences between box2dweb and box2dwasm
## how to test
[Uploading game.txt…]()
press q for applyForce, w for apply Impulse, e for setVelocity, r for return start point, u can check the difference between box2dweb and box2dwasm
